### PR TITLE
Graystudio Space65 ISO UK layout and keymap

### PR DESCRIPTION
**Pull request by rys**
ISO UK layout for the Graystudio Space65

## Description
Just a new `LAYOUT_65_iso` for the board and an implementation of it with no frills called `iso-uk`.

Tested on my personal Space65 built with that layout.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by fauxpark** [comment]
This doesn't seems like a UK-specific ISO layout, so it should just be named `iso` IMO. And yeah, `65_iso` should be added to the rules.mk `LAYOUTS`. The GH link in the readme shows the layout options:

File: keyboards/gray_studio/space65/keymaps/iso-uk/keymap.c [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [0] = LAYOUT_65_iso(
```
Comment (noroadsleft) [inline keyboards/gray_studio/space65/keymaps/iso-uk/keymap.c]:
```suggestion
    [0] = LAYOUT_65_iso_blocker(
```

File: keyboards/gray_studio/space65/keymaps/iso-uk/keymap.c [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+     * │Ctrl│LGUI│LAlt│      Spaaaaaaace       │RAlt│ L1 │ │ ← │ ↓ │ → │
+     * └────┴────┴────┴────────────────────────┴────┴────┘ └───┴───┴───┘
+     */
+        KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC, KC_GRV,
+        KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC,          KC_DEL,
+        KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT, KC_NUHS, KC_ENT,  KC_PGUP,
+        KC_LSFT, KC_NUBS, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT, KC_UP,   KC_PGDN,
+        KC_LCTL, KC_LGUI, KC_LALT,                   KC_SPC,                             KC_RALT, MO(1),            KC_LEFT, KC_DOWN, KC_RGHT
+    ),
+    [1] = LAYOUT_65_iso(
```
Comment (noroadsleft) [inline keyboards/gray_studio/space65/keymaps/iso-uk/keymap.c]:
```suggestion
    [1] = LAYOUT_65_iso_blocker(
```

File: keyboards/gray_studio/space65/space65.h [hunk 3]
```diff
@@ -55,3 +55,16 @@
     { k40, k41,   KC_NO, k43, KC_NO, KC_NO, KC_NO, k47, KC_NO, KC_NO, k4A, k4B, KC_NO, k4D,   k4E,   k4F }, \
 }
 
+#define LAYOUT_65_iso( \
```
Comment (noroadsleft) [inline keyboards/gray_studio/space65/space65.h]:
```suggestion
#define LAYOUT_65_iso_blocker( \
```

File: keyboards/gray_studio/space65/keymaps/iso-uk/readme.md [hunk 4]
```diff
@@ -0,0 +1,3 @@
+# ISO keymap for Space65
+
+Make this firmware with `make gray_studio/space65:iso-uk`.
```
Comment (noroadsleft) [inline keyboards/gray_studio/space65/keymaps/iso-uk/readme.md]:
```suggestion
Make this firmware with `make gray_studio/space65:iso`.
```

Comment (rys) [reply keyboards/gray_studio/space65/keymaps/iso-uk/readme.md]: Resolved by 805b505

File: keyboards/gray_studio/space65/keymaps/iso-uk/keymap.c (hunk 1 above)
Comment (rys) [reply keyboards/gray_studio/space65/keymaps/iso-uk/keymap.c]: Resolved by 805b505

**Comment by rys** [comment]
I made the requested changes in 805b505, thanks for the review. Learned a couple of things I'll definitely keep in mind for next time.

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**